src/install.rs

```diff
@@ -241,6 +241,27 @@ pub fn install(config: &InstallConfig) -> Result<()> {
         bail!("install failed");
     }
 
+    match get_filesystems_with_label("boot") {
+        Ok(pts) => {
+            let rootdev = std::fs::canonicalize(&config.device)
+                .unwrap_or_else(|_| PathBuf::from(&config.device))
+                .to_string_lossy()
+                .to_string();
+            let pts = pts
+                .iter()
+                .filter(|pt| !pt.contains(&rootdev))
```

Comment:
Using `contains` is probably okay, but is a bit sloppy.  In principle we could split out the first half of `mount_partition_by_label()` into a separate method, and use that to get the actual device path for the installed `boot` partition.  Not sure if it's worth it though.